src/Propel/Runtime/ServiceContainer/StandardServiceContainer.php

```diff
@@ -107,6 +108,11 @@ class StandardServiceContainer implements ServiceContainerInterface
      */
     protected $loggerConfigurations = [];
 
+    /**
+     * @var bool|null
+     */
+    protected $useDebugModeFlag;
```

Comment:
```suggestion
    protected $useDebugMode;
```
It is already a flag by design since it is bool
No need to mention that IMO.

Reply:
Yes, but `useDebugMode` is already taken by the method, and while I think it is possible to use the same name in a method and a variable, it probably leads to unnecessary confusion.

Naming it as the flag which is set by `useDebugMode()` seems like the most readable approach, but I am happy to change it. Let me know if you want to use the same name in both method and variable or prefer another name.

Reply:
Should we maybe use `$isInDebugMode` or alike?